server/gitrest/packages/gitrest-base/src/utils/helpers.ts

```diff
@@ -62,13 +75,23 @@ export function getRepoManagerParamsFromRequest(request: Request): IRepoManagerP
 	const storageRoutingId: IStorageRoutingId = parseStorageRoutingId(
 		request.get(Constants.StorageRoutingIdHeader),
 	);
+
+	const isEphemeralFromRequest = request.get(Constants.IsEphemeralContainer);
+	const isEphemeralContainer: boolean =
+		typeof isEphemeralFromRequest === "undefined"
```

Comment:
Curious: is there any difference between doing
`typeof isEphemeralFromRequest === "undefined"`
and 
`isEphemeralFromRequest === undefined`
?
From https://expressjs.com/en/api.html#req.get, it seems we would get the undefined type if a header is not present, so that made me think that `typeof` might be unnecessary.

Reply:
I just followed the way it is currently done for the initial query param. I vaguely remember that there was a livesite due to different versions treating the types differently. Perhaps @znewton remembers why we had to do this for initial?

Reply:
Simplified the logic as suggested